diff -ru kodi-15.1-f4dda26/addons/resource.language.en_gb/resources/strings.po kodi-15.1-f4dda26-new/addons/resource.language.en_gb/resources/strings.po
--- kodi-15.1-f4dda26/addons/resource.language.en_gb/resources/strings.po	2015-08-17 16:46:17.000000000 +0000
+++ kodi-15.1-f4dda26-new/addons/resource.language.en_gb/resources/strings.po	2015-09-19 09:13:52.889652775 +0000
@@ -10594,7 +10594,7 @@
 #: xbmc/dialogs/GUIDialogSmartPlaylistEditor.cpp
 #: xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeOverview.cpp
 msgctxt "#20343"
-msgid "TV shows"
+msgid "TV Shows"
 msgstr ""
 
 msgctxt "#20344"
diff -ru kodi-15.1-f4dda26/configure.ac kodi-15.1-f4dda26-new/configure.ac
--- kodi-15.1-f4dda26/configure.ac	2015-08-17 16:46:17.000000000 +0000
+++ kodi-15.1-f4dda26-new/configure.ac	2015-09-19 09:22:35.025652775 +0000
@@ -1051,7 +1051,7 @@
       AC_DEFINE([HAVE_LIBEGL],[1],["Define to 1 if you have the `EGL' library (-lEGL)."])
       AC_DEFINE([HAVE_LIBGLESV2],[1],["Define to 1 if you have the `GLESv2' library (-lGLESv2)."])
       AC_MSG_RESULT(== WARNING: OpenGLES support is assumed.)
-      LIBS="$LIBS -lEGL -lGLESv2 -lbcm_host -lvcos -lvchiq_arm -lmmal -lmmal_core -lmmal_util"
+      LIBS="$LIBS -lEGL -lGLESv2 -lbcm_host -lvcos -lvchiq_arm -lvcsm -lmmal -lmmal_core -lmmal_util"
     else
       AC_CHECK_LIB([EGL],   [main],, AC_MSG_ERROR($missing_library))
       AC_CHECK_LIB([GLESv2],[main],, AC_MSG_ERROR($missing_library))
